frontend/src/components/account-mgmt/ManageUserDialog.vue

```diff
@@ -382,6 +402,56 @@ export default {
         this.selectedFacilityIds = this.facilitiesToAdminister?.map((facility) => facility.facilityId)
       }
     },
+
+    /**
+     * Check if BCeID exists in provider organization, if so raise error message.
+     */
+    async checkBCeIDExists(userName) {
+      try {
+        if (this.user.userName) {
+          const res = await ApiService.apiAxios.get(`${ApiRoutes.USER}/${userName}?providerProfile=false`);
+          if (res.data.length >= 1) {
+            this.errorMessages = ['A user with this BCeID already exists.'];
+          } else {
+            this.errorMessages = [];
+          }
```

Comment:
You could do this pretty cleanly with a ternary expression.

Reply:
Fixed.